test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -806,6 +819,364 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
+				)).ToNot(HaveOccurred())
+
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+
+				By("Waiting when mcp finishes updates")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+
+				stalldEnabled, rtKernel := true, true
+				noHzParam := fmt.Sprintf("nohz_full=%s", *profile.Spec.CPU.Isolated)
+				sysctlMap := map[string]string{
+					"kernel.hung_task_timeout_secs": "600",
+					"kernel.nmi_watchdog":           "0",
+					"kernel.sched_rt_runtime_us":    "-1",
+					"vm.stat_interval":              "10",
+				}
+
+				kernelParameters := []string{noHzParam, "tsc=nowatchdog", "nosoftlockup", "nmi_watchdog=0", "mce=off", "skew_tick=1",
+					"processor.max_cstate=1", "intel_idle.max_cstate=0", "intel_pstate=disable", "idle=poll"}
+				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+
+				//Update the profile to disable HighPowerConsumption and enable PerPodPowerManagment
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption:  pointer.Bool(false),
+					RealTime:              pointer.Bool(true),
+					PerPodPowerManagement: pointer.BoolPtr(true),
+				}
+				if !*profile.Spec.RealTimeKernel.Enabled {
+					profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
+						Enabled: pointer.BoolPtr(true),
+					}
+				}
+
+				By("Patching the performance profile with workload hints")
+				newspec, err := json.Marshal(profile.Spec)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, newspec)),
+					),
+				)).ToNot(HaveOccurred())
+
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+
+				By("Waiting when mcp finishes updates")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+
+				stalldEnabled, rtKernel = true, true
+				noHzParam = fmt.Sprintf("nohz_full=%s", *profile.Spec.CPU.Isolated)
+				sysctlMap = map[string]string{
+					"kernel.hung_task_timeout_secs": "600",
+					"kernel.nmi_watchdog":           "0",
+					"kernel.sched_rt_runtime_us":    "-1",
+					"vm.stat_interval":              "10",
+				}
+
+				kernelParameters = []string{noHzParam, "tsc=nowatchdog", "nosoftlockup", "nmi_watchdog=0", "mce=off", "skew_tick=1", "intel_pstate=passive"}
+				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+
+			})
+
+			It("[test_id:54179]Verify System is tuned when reverting from PerPodPowerManagement to HighPowerConsumption", func() {
```

Comment:
And this one overrides this: https://github.com/marioferh/cluster-node-tuning-operator/blob/5d9c2e11ad5595b7724edb327ed1a34e83518fc1/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go#L676

Reply:
Yes, this test we are checking if we can move from one Hint to another is the system tuned appropriately